<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtVLCApp</class>
 <widget class="QWidget" name="QtVLCApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>251</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QLabel" name="lbBuild">
     <property name="text">
      <string>----</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="lbVersion">
     <property name="text">
      <string>----</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="bxSerial_2">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="2">
           <widget class="QComboBox" name="cbBaudRate">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
             </font>
            </property>
            <item>
             <property name="text">
              <string>9600</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>14400</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>57600</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>115200</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Port</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="cbSerialPort">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="btSerial">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Connect</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="font">
              <font>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Yaw</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="font">
              <font>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Pitch</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="font">
              <font>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Roll</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLineEdit" name="tbYaw">
             <property name="font">
              <font>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="tbPitch">
             <property name="font">
              <font>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="tbRoll">
             <property name="font">
              <font>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lbStatus">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>----</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
